presidio-anonymizer/presidio_anonymizer/operators/custom.py

```diff
@@ -0,0 +1,44 @@
+"""Replaces the PII text with lambda result."""
+from typing import Dict
+
+from presidio_anonymizer.operators import Operator, OperatorType
+from presidio_anonymizer.services.validators import validate_type
+from presidio_anonymizer.entities import InvalidParamException
+
+
+class Custom(Operator):
+    """
+    Replace PII text entity with the lambda result executed on the PII text.
+
+    lambda retrun type must be a string
+    If the new value is not a lambda, this will act as a replace method
+    """
+
+    NEW_VALUE = "new_value"
+
+    def operate(self, text: str = None, params: Dict = None) -> str:
+        """:return: new_value."""
+        new_val = params.get(self.NEW_VALUE)
+        if not new_val:
+            return f"<{params.get('entity_type')}>"

```

Comment:
You can call Relace.operate() to avoid duplication. Also, in other cases we assume that existence and validity are checked in validate().. so falling back to another implementation might be misleading (in case I accidentally  provided the function in a different variable than 'new_value' or for  an invalid statured function)

Reply:
yep, debating if to allow other values expect for a function. will make it simpler.

Reply:
Why are we falling back here?
might be the user wants empty values for some values and not empty for other no?

Reply:
I removed this. If the user wants to use any other method of anonymization, they can select a different operator. This one will only serve functions